Documentation/subcommands/export.md

```diff
@@ -0,0 +1,17 @@
+# rkt export
+
+Exports an exited, single-app pod to an App Container Image (.aci)
+
+```
+$ rkt export UUID .aci
+```
+
+## Options
+
+| Flag | Default | Options | Description |
+| --- | --- | --- | --- |
+| `--overwrite` |  `false` | `true` or `false` | Overwrite the output aci if it exists  |
```

Comment:
I have no issues with that. Personal preference, I thought it was easier to read